mflowgen/tile_array/Tile_MemCore/get_Tile_MemCore_outputs.sh

```diff
@@ -8,6 +8,8 @@ else
     make cadence-pegasus-lvs
 fi
 
+make pwr-aware-gls
+
```

Comment:
I don't think this should be automatically included in the flow, since it is not necessary to produce the outputs. We should add it to the CI tests instead.